### ISSUE TITLE
Kaiju Phylogenetic Tree to include in a Phyloseq object

### ISSUE THREAD

**Issue by magibc** [bioinformatics-centre/kaiju#215]
Hi!

In my first metagenomic study, I have used Kaiju v.1.8.2 for taxonomic classification and using Progenomes database.

Nevertheless, I would like to know if is there any availble kaiju progenomes phylogenetic tree? Or any idea/hints of how can I construct a phylogenetic tree to include it in a phyloseq object for downstream analysis?

Thanks a lot,

Magi.

**Comment by pmenzel** [comment]
Creating a phylogenetic tree from a dataset is a different concept than what kaiju is doing. Here, the tree is already given in the form of a cladogram (i.e. the NCBI taxonomy) into which the query sequences are placed.

**Comment by magibc** [comment]
Thank you @pmenzel for your quick response. 

Then from the taxonomical classification obtained from Kaiju, it is not possible to build a phylogenetic tree for downstream analyses? The cladogram that you comments could be downloaded from the Kaiju pipelines?

I know that Kaiju software tool purpose is not phylogenetic tree building but only I would like to know if you can give some hints/recommendations of which steps can I perform from Kaiju's output to build a phylogenetic tree? that I think that this type of question would also help fledging bioinformaticians as me starting in metagenomics field.

Thanks again,

Magi.

**Comment by pmenzel** [comment]
I don't know what kind of phylogenetic tree you would expect and what you want to do with it.
The cladogram is implicit in the structure of the NCBI taxomomy, but there is nothing to download here.

Normally, people use the ouput of metagenomic classifiers to summarize the relative abundance of, for example, species, genera or higher taxonomic ranks within a sample, by using the number of reads assigned to a taxon as a proxy for its abundance in the sample.

**Comment by pmenzel** [comment]
Yeah, I got that, but as I said: I wouldn't know how to make such a tree from shotgun metagenomic data.

**Comment by magibc** [comment]
Ok thank you Peter for your feedback. I appreciate it !

**Comment by magibc** [comment]
I would like to re-open this issue @pmenzel  in order to know if it is available from Kaiju (apart from kaiju_db_progenomes.fmi file which is too large) the references/code (if comes from NCBI taxon id...) for each of the protein fasta files used by Kaiju when I used proGenomes as database to run Kaiju.

I would need this information in order to download all the fasta references to build a custom tree to solve my problem discussed in the present issue.

Thanks again,

**Comment by magibc** [comment]
If someone could helps, I shared my way on how to construct a phylogenetic tree from Kaiju's using proGenomes as database:
https://bioinformatics.stackexchange.com/questions/19018/phylogenetic-tree-building-from-progenomes-database-for-shotgun-metagenomics/19124#19124
